console-api/proto/tasks.proto

```diff
@@ -76,8 +77,11 @@ message Task {
     // The location in code where the task was spawned.
     common.Location location = 6;
 
+    // The category of task this task belongs to.
     enum Kind {
+        // A task spawned via `tokio::task::spawn`.
         SPAWN = 0;
+        // A task spawned via `tokio::task::spawn_blocking`.
```

Comment:
it might be worth making this a little more generic since we would like to (eventually) support non-Tokio runtimes:
```suggestion
        // A task spawned using a runtime's standard asynchronous task spawning
        // operation (such as `tokio::task::spawn`).
        SPAWN = 0;
        // A task spawned via a runtime's blocking task spawning operation
        // (such as `tokio::task::spawn_blocking`).
```
or something like that?